interp/notation.mli

```diff
@@ -338,6 +338,8 @@ type level = notation_entry * entry_level * entry_relative_level list
 val level_eq : level -> level -> bool
 val entry_relative_level_eq : entry_relative_level -> entry_relative_level -> bool
 
+val notation_subentry_entry_level_lt : notation_subentry_level -> notation_entry_level -> bool
```

Comment:
This function was added in the API: it should be documented (even briefly).

Reply:
Thanks for asking. Actually, I made a mistake. What should be exported is [notation_subentry_entry_level_lt].

While trying to document it, I'd be tempted to say: "Tell if a subentry is a sub-entry of entry where subentry denotes the level of a variable of a notation and sub-entry tells that the level associated to the subentry is a sublevel of the level associated to the entry", which is a bit disturbing, so at the end, I'm wondering whether `subentry` is a well chosen terminology. Maybe, `notation_entry_level ` should be called `notation_rule_entry_level` and `notation_subentry_level` should be called `notation_variable_entry_level`?

Would the second look better for you?